pre-docs/bolt_command_reference.md

```diff
@@ -328,6 +328,22 @@ List modules available to Bolt.
 | `--boltdir FILEPATH` | Specify what Boltdir to load config from. | Autodiscovered from current working directory. |
 | `--configfile FILEPATH` | Specify where to load config from. | `~/.puppetlabs/bolt/bolt.yaml` |
 
+## `puppetfile generate-types`
```

Comment:
What's the reason for making this a separate command and not running it automatically as part of `bolt puppetfile install`?

Reply:
We talked about it in the meeting about this. I think it is so you do not *have* to have a puppetfile at all? Like maybe you are developing a new module and writing some types, you can just generate-types every time you want to refresh?

Reply:
That's a good reason to have a command for it. Is there a reason not to also do it in `bolt puppetfile install` though?

Reply:
I think that makes sense. Will update to do it automatically with puppetfile install.